### PR TITLE
Specification for Fabrica Trigger from Saleforce

### LINKED ISSUE #36: Create REST API records from Salesforce (Applikontech)
Specification: Create REST API member records from Salesforce

https://docs.google.com/document/d/1aBtrF3S9dyTdjTNUeul7t9qO3ObjWhcv-kBGoOh4UxE/edit#heading=h.tx7e3cf69dk4

### PR DESCRIPTION
**Pull request by kjgarza**
## Purpose

This is a new product specification. 
To review go to **Files changed** and click **Display the rich Diff**

closes: https://github.com/datacite/tasks/issues/36

## Types of changes

- [x] New feature (non-breaking change which adds functionality)

- [ ] Breaking change (feature that would cause existing functionality to change)

- [ ] New Product (non-breaking change which adds a new service)

## Reviewer, please remember to:

- Be humble in the language and feedback you give, ask don't tell.
- Consider using positive language as opposed to neutral when offering feedback. This is to avoid the negative bias that can occur with neutral language appearing negative.
- Ensure you give reasons for the changes you are proposing.

### CODE REVIEW

File: product_specifications/34_fdkhjfb5/readme.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+The REST API has all the functionality needed to manage members, repositories and soon also contacts via API calls. This is how Fabrica works, and an external developer could do exactly the same, e.g. when integrating with Salesforce.
+
+
+## User Stories
+
+### CRITICAL
+
+1. As a Saleforce Manager, I would like that when a Member application is approaved in Saleforce a Member account is generate in Fabrica (Production), So that I do not spend time on generating it manually.
+1. As a Saleforce Manager, I would like that when a Opportunity is created in Saleforce a Member account is generated in Fabrica (Stage), So that I do not spend time on generating it manually.
```
Comment (sarala) [inline product_specifications/34_fdkhjfb5/readme.md]: Why in Fabrica 'stage'? Is it not test?

File: product_specifications/34_fdkhjfb5/readme.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+             And the call should include Fabrica ID, date created, date last updated, and date deleted.
+
+        Scenario: Automatic Test Member Account Creation
+            Given that a opporunity has been created in Salesforce
+             When a opporunity has been created in Salesforce
+             Then a call to the Test instace API should be triggered https://api.test.datacite.org
+             And the call's payload should follow the REST API specification 
+             And the call should include Fabrica ID, date created, date last updated, and date deleted.
+```
+
```
Comment (sarala) [inline product_specifications/34_fdkhjfb5/readme.md:44]: I think we probably need another section which describe the relevant api calls and parameters. I think we can reference support documents but more detail here so external dev can work efficiently. What do you think?

Comment (kjgarza) [reply product_specifications/34_fdkhjfb5/readme.md:44]: Ok, yeah I can add that part

**Comment by kjgarza** [comment]
@sarala by the way, are email for billing contact and voting contact something that new members creation should pass to Fabrica? Or that is something that happens later in our business Model workflow?

**Review by sarala — APPROVED** [review]
All looks good

**Comment by sarala** [comment]
Yes, good point. I'm not sure about contact information. I wonder whether it is be based on the information here https://datacite.org/membership-enquiry.html

**Pull request merged**